staff/enric-pedros/precourse/bingo.js

```diff
@@ -0,0 +1,117 @@
+
+
+
+function usuario() { //nombre de usuario
+    var usuarionuevo = prompt("Buenos dias, indique su nombre: ");
+    return usuarionuevo;
+}
+
+var cartro = []
+var lineasCartro = 5

```

Comment:
Hey Enric ! I suggest you to end all the declarations with ";" just to have it all more clear! Just my point of view. Don't go to sleep too late :)

Reply:
Hey Bro! thx for this suggestion , i forgot the brand!